src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -97,6 +97,7 @@ const WorkspaceTabs = ({
       h(WorkspaceMenu, {
         iconSize: 27, popupLocation: 'bottom',
         callbacks: { onClone, onShare, onLock, onDelete, onLeave },
+        // TODO: this is where tabs are made visible
```

Comment:
Is this PR still WIP?

Will the Data tab be functional (no errors) for Azure workspaces?

Reply:
sorry, meant to add WIP to this!

Thanks for marking though

Reply:
FYI, you can open a PR as a draft. Draft PRs won't have review automatically requested from code owners and won't be posted in Slack until they are marked as "ready for review".

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests